cinema/serializers.py

```diff
@@ -0,0 +1,31 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=255)
+    description = serializers.CharField(required=False, allow_blank=True)
+    duration = serializers.IntegerField()
+
+    def validate_title(self, value):
+        old_title = None
+        if self.instance is not None:

```

Comment:
Using `if self.instance` is a more concise  way of checking for non-None values.